usaspending_api/api_contracts/contracts/v2/download/disaster.md

```diff
@@ -10,9 +10,8 @@ Creates a new download job for the requested account and award. Returns a link t
 + Request (application/json)
     + Attributes (object)
         + `filters` (optional, Filters, fixed-type)
-            Currently ignored, download will only be generated for COVID-19 spending data
         + `file_format` (optional, enum[string])
-            Currently ignored, download will only be generated with CSV data files
+            Currently only works for a download filtered on a single DEFC
```

Comment:
Should this have `Members` listed since it's an enum?
```suggestion
            The format of the file(s) in the zip file containing the data. Currently only works for a download filtered on a single DEFC
            + Default: `csv`
            + Members
                + `csv`
                + `tsv`
                + `pstxt`
```